packages/plugins/graphql-plugin-auth/src/Neo4jGraphQLAuthJWTPlugin.ts

```diff
@@ -65,6 +74,10 @@ class Neo4jGraphQLAuthJWTPlugin {
         return result;
     }
     /* eslint-enable @typescript-eslint/require-await */
+
+    public getGlobalAuthenticationEnabled(): boolean {
+        return this.globalAuthentication;
+    }
```

Comment:
Mmm, I'm not too fond of this duplicate code between both plugins. Maybe we could create a new abstract class `AuthPlugin` inherited by these 2 plugins so we can share this method and property?

Reply:
Good point!